.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*   @s8sato @mversic @arjentix @SamHSmith @QuentinI @outoftardis @ilchu @pesterev @Erigara @0x009922 @DCNick3
+*   @s8sato @mversic @arjentix @SamHSmith @outoftardis @Erigara @0x009922 @DCNick3 @vamuzing @Asem-Abdelhady @horizenight
```

Comment:
who is `horisenight`?

Reply:
Kshitij Roodkee, who worked on the Blockchain Explorer and later was present in a company for some time. He plans to remain a contributor.